src/app/build.gradle

```diff
@@ -1,88 +1,87 @@
 buildscript {
-	ext.kotlin_version = '1.3.30'
+    ext.kotlin_version = '1.3.30'
```

Comment:
this file hasn't actually changed, I just accidentally formatted it and changed all the indentation.